Guidance to not use forwarding to abort processing (#5819)
Particular/docs.particular.net: nservicebus/messaging/forwarding.md

Use this feature to forward successfully processed messages from an endpoint to a specific destination endpoint. Forwarding messages is particularly useful in complex upgrade scenarios when the old version and new version of a particular endpoint are running side-by-side.

> ## Abort processing

Reviewer: ```suggestion
## Message processing abortion
```

> Forwarding should not be used to abort processing.  Messages cannot be forwarded to the `error` queue as no meta data would be present in the message meta data.  Forwarding to abort processing can result in partial updates.

Reviewer 2: ```suggestion
Forwarding should not be used to abort message processing. Messages cannot be forwarded to the `error` queue as the required metadata for failed messages would not be present in the forwarded message's metadata.  Attempting to use forwarding as a mechanism to abort processing can result in partial updates.
```

> Forwarding should not be used to abort processing.  Messages cannot be forwarded to the `error` queue as no meta data would be present in the message meta data.  Forwarding to abort processing can result in partial updates.

Reviewer: ```suggestion
Forwarding should not be used to abort message processing. Messages cannot be forwarded to the `error` queue as no metadata would be present in the message metadata. Forwarding to abort message processing can result in partial updates.
```

> To abort processing throw an exception which ensures no ghost message are transmitted and any (transactional) state to be rolled back. 

Reviewer 2: ```suggestion
To abort processing throw an exception in the message handler. This ensures that no ghost message are transmitted and any transactional state changes are rolled back. 
```

> To abort processing throw an exception which ensures no ghost message are transmitted and any (transactional) state to be rolled back. 

Reviewer: ```suggestion
To abort message processing, throw an exception that ensures no ghost messages are transmitted, and any (transactional) state will be rolled back. 
```

> To prevent unnessary retries declare a [unrecoreable exception](/nservicebus/recoverability/#unrecoverable-exceptions) or create a [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md).

Reviewer: ```suggestion
To prevent unnessary retries declare an [unrecoreable exception](/nservicebus/recoverability/#unrecoverable-exceptions) or create a [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md).
```
